rs/replicated_state/src/canister_state/tests.rs

```diff
@@ -267,6 +267,29 @@ fn canister_state_push_input_best_effort_response_no_reserved_slot() {
     assert!(!fixture.canister_state.has_input());
 }
 
+#[test]
+fn canister_state_push_input_best_effort_response_canister_stopped() {
+    let mut fixture = CanisterStateFixture::new();
+
+    fixture.with_input_slot_reservation();
+    let response = default_input_response(fixture.make_callback(SOME_DEADLINE), SOME_DEADLINE);
+
+    // Stop the canister.
+    fixture
+        .canister_state
+        .system_state
+        .set_status(CanisterStatus::Stopped);
+
+    // The best-effort response should be dropped silently.
+    assert!(!fixture
+        .push_input(
+            response.into(),
+            SubnetType::Application,
+            InputQueueType::RemoteSubnet
+        )
+        .unwrap());
```

Comment:
Nit: This would be more readable as:

```suggestion
    assert_eq!(
        Ok(false),
        fixture
            .push_input(
                response.into(),
                SubnetType::Application,
                InputQueueType::RemoteSubnet
            )
    );
```